app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/common/AlertDialogs.kt

```diff
@@ -0,0 +1,52 @@
+package com.kks.nimblesurveyjetpackcompose.ui.presentation.common
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import com.kks.nimblesurveyjetpackcompose.model.ErrorModel
+import com.kks.nimblesurveyjetpackcompose.ui.theme.CuriousBlue
+
+@Composable
+fun ErrorAlertDialog(
+    errorState: ErrorModel,
```

Comment:
Shall we rename this to `errorModel` for parity?
```suggestion
    errorModel: ErrorModel,
```